### DISCUSSION TITLE
Welcome to damaihelper Discussions!

### DISCUSSION THREAD

**Discussion by Guyungy** [Guyungy/damaihelper#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GuoyuZhaoPSY** [comment]
😇我就是用这个脚本抢深圳场的票，本来都抢到了结果无法提交订单

**Reply by Guyungy** [reply]
我也是

**Reply by zwznzkl** [reply]
有抢成功的人吗?我写不来脚本啊,有偿帮抢的有没有

**Comment by chenjiangcai** [comment]
大佬可以请教一下吗，现在这个脚本还可以用吗，那些设置我有点不懂

**Comment by kcd2333** [comment]
大佬我已经配置好了，代码也可以跑起来，但是原本购买的按钮变成了该渠道无法购买，是不是我被检测出来了啊，我要怎么躲过这个检测

**Reply by kcd2333** [reply]
![Uploading 截屏2024-04-29 18.12.43.png…]()

**Comment by xiqing2002** [comment]
我配置脚本一直启动不了，配置问题可以帮我一下嘛？
